po/it.po

```diff
@@ -391,12 +402,10 @@ msgid "Starting new chats is currently broken in Axolotl."
 msgstr "L'avvio di nuove chat è attualmente interrotto in Axolotl."
 
 #: src/pages/About.vue:6
-#, fuzzy
 msgid ""
 "This is a free and open source Signal client written in golang and vuejs."
 msgstr ""
-"Questo è un client di signal gratuito e open source scritto in golang e "
-"vuejs."
+Questo è un client Signal gratuito e open source scritto in golang e vuejs.
```

Comment:
Thanks for your translation, this has to be in "" before we can merge it.

Reply:
Done!